decode/render.ts

```diff
@@ -94,25 +94,10 @@ function box(id: number, region: string, rectangle: number[], iframe: HTMLIFrame
             doc.body.appendChild(layer);
             layer.innerText = region;
             nodes[id] = layer;
-        } else if (el && el.tagName === "IFRAME") {
-            let s = getComputedStyle(el, null);
-            let width = rectangle[2];
```

Comment:
This is the change that removes iframes from the box model data and only sends for regions?

Reply:
Yes that's right.